package.json

```diff
@@ -4,9 +4,10 @@
   "description": "",
   "main": "server.js",
   "scripts": {
-    "start": "nodemon server/server.js",
-    "dev": "concurrently \"cross-env NODE_ENV='development' webpack-dev-server --history-api-fallback \" \"nodemon ./server/server.js\"",
+    "start": "node ./server_dist/server.js",
```

Comment:
`start` is usually what's used by host to initiate the server. This should not be nodemon.